style.css

```diff
@@ -0,0 +1,211 @@
+body
+{
+    background-color: rgb(114, 113, 113);
+
+}
+#container
+{
+    width: 1000px;
+    margin-left: auto;
+    margin-right: auto;
+    margin-top: 20px;
+
+}
+
+#logo
+{
+    color: white;
+    font-size: 38px;
+    letter-spacing: 2px;
+    padding: 10px;
+    text-align: center;
+    font-weight: 900;
+    
+}
+
+#menuT
+{
+    margin-top: 20px;
+    background-color: black;
+    color: white;
+    font-size: 20px;
+    padding: 15px;
+    margin-bottom: 10px;
+}
+
+#OW
+{
+    background-color:orange;
+    font-size: 20px;
+    opacity: 0.6;
+
+}
+
+#tytul
+{
+    text-align: center;
+    font-size: 30px;
+    padding: 10px;
+    border-bottom: 3px dotted black;
+    letter-spacing: 2px;
+}
+
+
+#wyjasnienie
+{
+    text-align: justify;
+
+
+}
+
+
+#menuL
+{
+  float: left; 
+  margin-top: 10px;
+  text-align: center; 
+  height: 300px;
+  
+
+}
+
+.optionML
+{
+    min-width: 280px;  
+    border-bottom: 3px dotted gray;
+    background-color: black;
+    color: white;
+    min-height: 53px;
+    padding-top: 30px;
+
+}
+
+.optionML:hover
+{
+    background-color: white;
+    color: black;
+    border-bottom: 3px dotted black;
+    cursor: pointer;
+
+}
+
+#article
+{
+    float: left;
+    width: 670px;
+    padding: 20px;
+    background-color: orange;
+    opacity: 0.6;
+    margin-top: 10px;
+    margin-left: 10px;
+    text-align: center;
+    height: 300px;
+}
+
+#co
+{
+    font-size: 29px;
+    border-bottom: 3px dotted black;
+    padding: 15px;
+    font-weight: 900;
+
+}
+
+#na
+{
+    font-size: 25px;
+    padding: 10px;
+}
+
+#footer
+{
+    clear: both;
+    background-color: black;
+    color: white;
+    font-size: 15px;
+    padding: 10px;
+    text-align: center;
+    
+
+}
+
+
+.option
+{
+    float: left;
+    min-width: 50px;
+    margin-left: 10px;
+    margin-right: 10px;
+    border-right: 3px dotted gray;
+    padding: 10px;
+
+}
+
+.option:hover
+{
+    background-color: white;
+    color: black;
+    border-right: 3px dotted black;
+    cursor: pointer;
+
+}
+
+.descripe
+{
+    width: 1000px;
+    margin-top: 10px;
+    margin-bottom: 50px;
+    
+}
+
+.imie
+{
+    width: 980px;
+    height: 40px;
+    background-color: black;
+    color: whitesmoke;
+    margin-top: 10px;
+    margin-bottom: 10px;
+    text-align: center;
+    font-size: 30px;
+    padding: 10px;
+}
+
+.obraz
+{
+    float: left;
+    height: 310px;
+    width: 180px;
+    border: 2px dotted whitesmoke;
+}
+
+.przypis
+{
+    float: left;
+    width: 782px;
+    padding: 10px;
+    text-align: justify;
+    background-color: rgb(114, 113, 113);
+    color: whitesmoke;
+    font-size: 25px;
+    border: 2px dotted whitesmoke;
+    margin-left: 10px;
+}
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
```

Comment:
when writing a code, try to make it pretty as much as possible ;) This is super highly important, because readability is one the most important things for developers. Most of the time, we are reading somoene else code... If it is not readable we are slow.